lib/kitchen/directions/bake_chapter_references/v1.rb

```diff
@@ -4,11 +4,12 @@ module Kitchen::Directions::BakeChapterReferences
   class V1
     renderable
 
-    def bake(chapter:, metadata_source:, uuid_prefix: '.', klass: 'references')
+    def bake(chapter:, metadata_source:, uuid_prefix: '.', klass: 'references', from_introduction: false)
       @metadata = metadata_source.children_to_keep.copy
       @klass = klass
       @title = I18n.t(:references)
       @uuid_prefix = uuid_prefix
+      @from_introduction = from_introduction
 
       chapter.references.search('h3').trash
```

Comment:
again regarding this line, along with line 20 & 34 ... i believe if you change this to `chapter.references.search('h3').each(&:trash)` it will trash all `h3` tags within a reference. if this is the behavior you're going for, i'd favor this approach over calling this line again on lines 20 & 34

Reply:
Yes, this gives the expected result. Thank you Kendra! :slightly_smiling_face: